homeassistant/components/freebox/strings.json

```diff
@@ -15,11 +15,11 @@
     },
     "error": {
       "register_failed": "Failed to register, please try again",
-      "connection_failed": "Failed to connect, please try again",
-      "unknown": "Unknown error: please retry later"
+      "connection_failed": "[%key:common::config_flow::error::cannot_connect%]",
```

Comment:
It's good to also replace the key with a standard one, eg `"cannot_connect"` and update the config flow and tests.

Reply:
Will do in another PR